src/lib/CombatCalc.ts

```diff
@@ -57,6 +57,24 @@ const ONE_HIT_MONSTERS: number[] = [
   11193, // Flower (A Night at the Theatre)
 ];
 
+const THRALL_DPT = 0.375;
+const THRALL_SPEED = 4;
```

Comment:
I like to preserve the math in constants so it's more clear how they were derived:
```ts
const THRALL_MAX = 3;
const THRALL_SPEED = 4;
const THRALL_DPT = sum(range(0, THRALL_MAX + 1)) / ((THRALL_MAX + 1) * THRALL_SPEED);
```